### DISCUSSION TITLE
Why Doesn't Tweaks hide the Dashboard Carousel?

### DISCUSSION THREAD

**Discussion by ghost** [AprilSylph/XKit-Rewritten#680] [Suggestion Box]
I would personally like the option to, since I never use any of the links as is. Not having to see it would honestly be a big improvement.

I feel like I remember there being a reason for this, so if there is I would be interested in hearing it! Thank you. :)

**Comment by AprilSylph** [comment]
When not using Best Stuff First, it serves as an indicator for where the previous browsing session started. The carousel part is hidden when using the `Turn "You're caught up" into a separating line` option.

**Reply by ghost** [reply]
Ah, egg on my face lol. Thank you for the clarification, I appreciate it!
